client/src/document/ingredients/browser-compatibility-table/index.scss

```
@@ -140,6 +152,32 @@
 .bc-version-label {
   display: none;
 }
+.expert .bc-version-label {
+  display: initial;
+}
+
+.expert .bc-icon-cell {
+  > .icon-wrap {
+    display: block;
+  }
+}
+
+.expert {
```

For reasons of consistency, KISS and DRY, I would write this as follows:

```suggestion
.expert {
  & .bc-version-label {
    display: initial;
  }

  & .bc-icon-cell > .icon-wrap {
    display: block;
  }

```